src/cobald/decorator/logger.py

```diff
@@ -24,11 +24,13 @@ class Logger(PoolDecorator):
     ``value``
         for the new demand to set,
 
-    ``demand``, ``supply``, ``utilisation`` and ``consumption``
+    ``demand``, ``supply``, ``utilisation`` and ``allocation``
         for the current state of ``target``, and
 
     ``target``
         for the raw ``target`` pool.
+
+    For historical reasons, ``consumption`` is available as an alias of ``allocation``.
```

Comment:
Does this mean, that there is still COBalD code around using `consumption` or is this for backward compatibilty of other plugins? Do we need this?

Reply:
Maybe we should put a deprecation warning here?

Reply:
I simply don't know if anyone is using it. Wouldn't mind dropping it, though.

A deprecation warning should be possible as well.

Reply:
+1 for a deprecation warning.

Reply:
I've started working on deprecation warnings, but it seems this will be a major feature to do right – we need a ``DeprecatedKeyMapping`` (for this case), a deprecation version range (similar to [deprecation package](https://pypi.org/project/deprecation/)) and version support (probably [semantic_version](https://pypi.org/project/semantic-version/)) *at minimum*. A quick survey shows most of this must be custom (e.g. we're not deprecating a function but a key, so the ``deprecation`` package does not apply).

Since we're likely to deprecate lots of things in the future, I'd like to "do this right". This makes it too large to sneakily include in this PR, IMO.

@giffels , @eileen-kuehn , what is your opinion on how to proceed? We could either merge this and open a ticket to deprecate this (or not close #74), or I'll add the general deprecation support first in another PR and this PR gets done afterwards.

Reply:
I vote for creating a ticket to already merge this and improve later.